tracer/src/Datadog.Trace/Ci/CiEnvironment/GitInfo.cs

```diff
@@ -174,19 +197,11 @@ public static IGitInfo GetCurrent()
                 Log.Error(ex, "Get directories failed with UnauthorizedAccessException");
                 return null;
             }
-
-            if (gitDirectories.Length > 0)
+            catch (SecurityException ex)
             {
-                foreach (var gitDir in gitDirectories)
-                {
-                    if (gitDir.Name == ".git")
-                    {
-                        return gitDir;
-                    }
-                }
+                Log.Error(ex, "Get directories or parent directory failed with SecurityException");
```

Comment:
I don't think it's useful to report this error?

```suggestion
                Log.Warning(ex, "Get directories or parent directory failed with SecurityException");
```